lib/utils.js

```diff
@@ -64,11 +66,39 @@ const utils = {
     utils.log(`${c.redBright('Error')} ${err && err.message ? err.message : err}`, newline);
   },
 
-  loadUserConfig: async () => {
+  loadUserConfig: async (checkIsEmpty = true) => {
     const f = resolveUserConfig();
     await ensureFile(f);
     const config = yaml.safeLoad(await readFile(f)) || {};
-    if (isEmpty(config)) {
+    if (config.apiToken) {
+      const token = config.apiToken;
+      delete config.apiToken;
+      const decoded = jwt.decode(token, { complete: true }) || {};
+      if (decoded.payload && decoded.payload.iss) {
+        if (decoded.payload.iss === 'https://api.getstructure.io' || decoded.payload.iss === 'http://api.getstructure.io') {
+          decoded.payload.iss = 'https://api.losant.com';
+        }
+        const defaultUrl = decoded.payload.iss;
+        const api = await getApi({ apiToken: token, url: defaultUrl });
+        let wlInfo;
+        try {
+          wlInfo = await api.request({ method: 'get', url: '/whitelabels/domain' });
+        } catch (e) {
+          // could not get the white-label...
+          // just throw away the token they need to re login.
+        }
+        if (wlInfo) {
+          config[defaultUrl] = {
+            apiToken: token,
+            endpointDomain: wlInfo.endpointDomain,
+            appUrl: wlInfo.appUrl
+          };
+          await writeFile(f, yaml.safeDump(config));
+          return config;
+        }
+      }
+    }
+    if (checkIsEmpty && isEmpty(config)) {
```

Comment:
I don't think it makes a difference, but I like this check better *above* the migration check?

Reply:
wouldnt it if the token is invalid?